src/server.ts

```diff
@@ -51,6 +51,12 @@ export default async (): Promise<Express> => {
   app.use(bodyParser.urlencoded({ extended: true }))
   app.use(bodyParser.json())
   app.use(cors())
+  app.use((req, _, next) => {
+    // make sure we always have a file object on req even if this is not a multipart body this is so that
+    // the attachment route can handle both JSON and multipart bodies
```

Comment:
What do you think , should we write it like this ( I think this makes this block easier to read ) ...

```suggestion
    // make sure we always have a file object on req even if this is not a multipart body
    // this is so that the attachment route can handle both JSON and multipart bodies
```

How would it work if you have both JSON and file ( lol, i need to try this ) ?

Reply:
I'm going to dismiss this and get this merged. I get the point but this isn't worth holding this PR up any longer